model/authorization/TestTakerAuthorizationService.php

```diff
@@ -68,15 +69,27 @@ public function verifyResumeAuthorization(DeliveryExecution $deliveryExecution,
     }
 
     /**
-     * Whenever or not a delivery execution for a given delivery
-     * should be proctored
+     * Check if delivery id proctored
      *
      * @param string $deliveryId
      * @return boolean
      */
     public function isProctored($deliveryId, User $user)
     {
-        return !($user instanceof GuestTestUser);
+        $propertyUri = null;
+
+        if ($deliveryId) {
+            $delivery = new \core_kernel_classes_Resource($deliveryId);
```

Comment:
Use `\oat\generis\model\OntologyAwareTrait` to instantiate resources/classes/properties.